TypeCobol.Test/Parser/Programs/Cobol85/DebugNormalLines.rdz.cbl

```diff
@@ -57,6 +63,7 @@
            MOVE xxxx TO yyyy
       d    DISPLAY 'name = '
                 yyyy
+      d    DISPLAY 'name = ' yyyy
            MOVE '1234' TO yyyy
            GOBACK.
        END PROGRAM Pgm2.
```

Comment:
@delevoye 
Avoir un second test NoDebugNormalLines.rdz.cbl mais avec l'option debugging en commentaire (mettre en commentaire la ligne source-computer). Normalement on ne devrait plus avoir les diagnostics.

Reply:
Ok as seen with @delevoye the test is already present.